### PR TITLE
Add vulnerability search to OSS List

### LINKED ISSUE #939: OSS List > search with vulnerability
OSS List 내 검색 창 조건에서 vulnerability 추가

- 검색 조건 명 : Vulnerability (>=)
- input : 숫자
- 입력된 input 값보다 vulnerability score가 높은 경우 검색

Adding vulnerability in the search window in the OSS List menu.

- Search condition name: Vulnerability (>=)
- input : number
- Search if the vulnerability score is higher than the input value entered

### PR DESCRIPTION
**Pull request by jiwon83**
## Description
closes #939 
Add search condition in OSS List. 
Now, a user can search the OSS List by more than the vulnerability(>=) condition. And in the User Settings tab, the user can set user settings with vulnerability.

## Solution
- Add input tag in 2 JSP file( admin/oss/list.jsp and searchConditionArea.jsp)
```
<label>Vulnerability (>=)</label>
<input name="cvssScore" type="text" value="${searchBean.cvssScore}" style="width:230px"/>
```

<br>

- Add search query in OSSMapper.xml

```
<if test="!@oss.fosslight.util.StringUtil@isEmpty(cvssScore)">
	AND CAST(T1.CVSS_SCORE AS FLOAT ) >= CAST(#{cvssScore} AS FLOAT )
</if>
```

As Is :
Untitled (5)
<br>

To Be :
Untitled (7)
Untitled (8)
<br>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/resources/oss/fosslight/repository/OssMapper.xml [hunk 1]
```diff
@@ -658,6 +661,14 @@
 			)
 		</if>
 		<if test="@oss.fosslight.util.StringUtil@notEquals(versionCheck,'N')">
+
+
```
Comment (70825) [inline src/main/resources/oss/fosslight/repository/OssMapper.xml:665]: Please delete empty lines

**Review by 70825 — CHANGES_REQUESTED** [review]
Please edit the title of your pull request. Thank you

**Comment by MyunghyunNero** [comment]
I think you need to change >= to <![CDATA[>=]]> in xml. 
and it would be a good idea to look for it in [CDATA]

File: src/main/resources/oss/fosslight/repository/OssMapper.xml (hunk 1 above)
Comment (jiwon83) [reply src/main/resources/oss/fosslight/repository/OssMapper.xml:665]: Thank you for your delicacy. I resolve this!

**Comment by jiwon83** [comment]
> I think you need to change >= to "<![CDATA[>=]]>" in xml. and it would be a good idea to look for it in [CDATA]

@MyunghyunNero I modified it! I can learn about XML parser problems thanks to you.

**Comment by soimkim** [comment]
@jiwon83 , OSS List에서 검색시 Vulnerability 조건에 해당되지 않는 버전도 포함되는 버그가 있습니다.
image
해당 버그가 개선되면 더더욱 좋겠습니다. 😃

**Comment by soimkim** [comment]
위 버그는 https://github.com/fosslight/fosslight/issues/992에 이슈 등록해두었습니다.

**Pull request merged**